http://sourceware.org/ml/gdb-cvs/2012-05/msg00060.html

### src/gdb/doc/ChangeLog	2012/05/09 18:15:58	1.1310
### src/gdb/doc/ChangeLog	2012/05/09 19:29:25	1.1311
## -3,6 +3,9 @@
 	* gdb.texinfo (Auto-loading): Wrap too long lines in @smallexample.
 	Twice.
 
+	* gdb.texinfo (Separate Debug Files, Auto-loading safe path): Replace
+	directory separator by path separator.
+
 2012-05-06  Jan Kratochvil  <jan.kratochvil@redhat.com>
 
 	* gdb.texinfo (Auto-loading safe path): Make 'directories'
--- src/gdb/doc/gdb.texinfo	2012/05/09 18:15:58	1.956
+++ src/gdb/doc/gdb.texinfo	2012/05/09 19:29:25	1.957
@@ -16285,8 +16285,8 @@
 @kindex set debug-file-directory
 @item set debug-file-directory @var{directories}
 Set the directories which @value{GDBN} searches for separate debugging
-information files to @var{directory}.  Multiple directory components can be set
-concatenating them by a directory separator.
+information files to @var{directory}.  Multiple path components can be set
+concatenating them by a path separator.
 
 @kindex show debug-file-directory
 @item show debug-file-directory
@@ -21227,7 +21227,7 @@
 If you omit @var{directories}, @samp{auto-load safe-path} will be reset to
 its default value as specified during @value{GDBN} compilation.
 
-The list of directories uses directory separator (@samp{:} on GNU and Unix
+The list of directories uses path separator (@samp{:} on GNU and Unix
 systems, @samp{;} on MS-Windows and MS-DOS) to separate directories, similarly
 to the @env{PATH} environment variable.
 
@@ -21242,7 +21242,7 @@
 @item add-auto-load-safe-path
 Add an entry (or list of entries) the list of directories trusted for automatic
 loading and execution of scripts.  Multiple entries may be delimited by the
-host platform directory separator in use.
+host platform path separator in use.
 @end table
 
 Setting this variable to @file{/} disables this security protection.
